examples/text-generation/utils.py

```diff
@@ -349,9 +349,10 @@ def setup_model(args, model_dtype, model_kwargs, logger):
 
     if args.torch_compile:
         model = get_torch_compiled_model(model, logger, args)
-        assert "PT_HPU_LAZY_MODE" in os.environ and os.environ["PT_HPU_LAZY_MODE"] != "1", (
-            "`--torch_compile` is not compatible with PT_HPU_LAZY_MODE=1. Please set it to 0 or unset the variable."
-        )
+        if "PT_HPU_LAZY_MODE" in os.environ:
```

Comment:
@AKloniecki should you add this check before compiling the model?

Reply:
good point.
@AKloniecki is OOO, I will handle this PR.